test/soap.test.js

```diff
@@ -58,35 +52,25 @@ describe('soap', function () {
     });
     it('retrieveBulk: should return 2 data extensions', async function () {
         //given
-        mock.onPost(
-            '/Service.asmx',
-            {
-                asymmetricMatch: XMLValidator.validate,
-            },
-            {
-                Accept: 'application/json, text/plain, */*',
-                'Content-Type': 'text/xml',
-                SOAPAction: resources.retrieveBulkDataExtension.action,
-            }
-        )
+        mock.onPost('/Service.asmx', {
+            asymmetricMatch: XMLValidator.validate,
+        })
             .replyOnce(
                 resources.retrieveBulkDataExtension.status,
-                resources.retrieveBulkDataExtension.response
-            )
-            .onPost(
-                '/Service.asmx',
-                {
-                    asymmetricMatch: XMLValidator.validate,
-                },
+                resources.retrieveBulkDataExtension.response,
                 {
-                    Accept: 'application/json, text/plain, */*',
-                    'Content-Type': 'text/xml',
-                    SOAPAction: resources.retrieveDataExtension.action,
```

Comment:
SOAPAction is no longer needed?

Reply:
thought it was not relevant, but added back - requests a change in handling with the setup for new axios version